test/Confluent.Kafka.UnitTests/TopicPartitionError.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using Xunit;
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.UnitTests
+{
+    public class TopicPartitionErrorTests
+    {
+        [Fact]
+        public void Constuctor()
+        {
+            var tpe = new TopicPartitionError("mytopic", 42, ErrorCode.Local_BadMsg);
+
+            Assert.Equal(tpe.Topic, "mytopic");
+            Assert.Equal(tpe.Partition, 42);
+            Assert.Equal(tpe.Error, new Error(ErrorCode.Local_BadMsg));
```

Comment:
Might want to add a negative test here, tpe.Error is not equal to some other error code.